main/res/values/preference_keys.xml

```diff
@@ -190,6 +190,7 @@
     <string translatable="false" name="pref_persistablefolder_offlinemaps">persistablefolder_offlinemaps</string>
     <string translatable="false" name="pref_fakekey_info_offline_mapthemes">fakekey_info_offline_mapthemes</string>
     <string translatable="false" name="pref_persistablefolder_offlinemapthemes">persistablefolder_offlinemapthemes</string>
+    <string translatable="false" name="pref_persistablefolder_offlinemapshading">pref_persistablefolder_offlinemapshading</string>
```

Comment:
Please do not use "pref_" prefix for the preference variable value (only for preference variable name).